apps/bfd-server/bfd-server-war/src/test/resources/openapi/posts.yaml

```diff
@@ -351,6 +349,232 @@ paths:
+                        - `mbi=1EG4TE5MK73` (unhashed MBI)
+                        - `mbi=1S00E00HA26` (synthetic MBI)
+                    example: "1EG4TE5MK73"
+                  _lastUpdated:
+                    type: string
+                    description: |-
+                      Only satisfy the Search if the Beneficiary's `last_updated` Date falls within a specified _DateRange_.
+                      A _DateRange_ can be defined by providing less than `lt` and/or greater than `gt` values.
+                      This parameter can be included in a request one or more times.
+
+                      Examples:
+                        - `_lastUpdated=gt2023-01-02&_lastUpdated=lt2023-05-01` defines a range between two provided dates
+                        - `_lastUpdated=gt2023-01-02` defines a range between the provided date and today
+                        - `_lastUpdated=lt2023-05-01` defines a range from the earliest available records until the provided date
+                    example: "gt2023-01-02"
+                  excludeSAMHSA:
+                    type: string
+                    description: |-
+                      The _Substance Abuse and Mental Health Services Administration_ (SAMHSA)
+                      is the agency within the U.S. Department of HHS that leads public health efforts to advance the behavioral health of the nation.
+                      Setting this flag to _true_, modifies the request to filter out all SAMSHA-related claims from the response.
+                      _ClaimResponse_ doesn't contain any SAMHSA potential data. Due to the 1:1 relationship between _ClaimResponse_
+                      and _Claim_, any ClaimResponse resource will be excluded when the associated Claim resource contains SAMHSA data.
+
+                      Example:
+                        - `excludeSAMHSA=true`
+                    example: true
+                  includeTaxNumbers:
+                    type: string
+                    description: |-
+                      Setting this flag to _true_, provides tax number in the Claim transformed data for the response.
+                      
+                      Example:
+                         - `includeTaxNumbers=true
+                  isHashed:
+                    type: string
+                    description: |-
+                      Set this flag to false during POST request. Not setting the flag is defaulted to true
+                      
+                      Example:
+                        - `isHashed=true`
+                  service-date:
+                    type: string
+                    description: |-
+                      Only satisfy the Search request if a claim's Date
+                      falls within a specified _DateRange_. A _DateRange_ can be
+                      defined by providing less than `lt` and/or greater than `gt` values.
+                      This parameter can be included in a request one or more times.
```

Comment:
Do we know _which_ date this is referring to? Does it apply to just a subset, e.g. FISS vs MCS?

Reply:
description updated to : The _DateRange_ is defined by the claim's billing period, specifically the `Claim.billablePeriod.end`.